sympy/stats/crv.py

```diff
@@ -323,6 +324,22 @@ class ContinuousDistributionHandmade(SingleContinuousDistribution):
     def set(self):
         return self.args[1]
 
+    def _sample_scipy(self, size):
+        handmade_pdf = self.pdf
+        scipy = import_module('scipy')
+        from scipy.stats import rv_continuous
+        class scipy_pdf(rv_continuous):
+            def _pdf(self, z):
+                return handmade_pdf(z)
+
+        scipy_rv = scipy_pdf(a=self.set._inf, b=self.set._sup, name='scipy_pdf')
+
+        return scipy_rv.rvs(size=size)
+
+    def sample(self, size=()):
+        icdf = self._inverse_cdf_expression()
+        return icdf(random.uniform(0, 1))
```

Comment:
This should be `_sample_python`. `sample` should simply link to `_sample_python`.